dbt/models/proximity/columns.md

```diff
@@ -181,6 +181,21 @@ Nearest golf course ID, either Cook County ID or OSM ID.
 Golf course data sourced from Cook County GIS and OpenStreetMap
 {% enddocs %}
 
+## nearest_permit_dist_ft
+
+{% docs column_nearest_permit_dist_ft %}
+Nearest home improvement exemption distance (feet).
+
+Permit data sourced from Cook County GIS and IasWorld
+{% enddocs %}
+
+## nearest_permit_pin10
```

Comment:
I kept the identifying pin10 for the permit / construction since we don't have an OSMID.